Reference/global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "2.2.402"
```

Comment:
.net core 2.2 has reached EOL in December already and PSSA has already moved on to 3.1 in the root directory. I think we can simply remove the file